src/features/events/components/EventPopper/ParticipantAvatars.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Avatar, AvatarGroup, Box } from '@mui/material';
+
+import theme from 'theme';
+import ZUIPersonHoverCard from 'zui/ZUIPersonHoverCard';
+import {
+  ZetkinEventParticipant,
+  ZetkinEventResponse,
+} from 'utils/types/zetkin';
+
+interface ParticipantAvatarsProps {
+  orgId: number;
+  participants: ZetkinEventParticipant[] | ZetkinEventResponse[];
+}
+
+const ParticipantAvatars = ({
+  participants: list,
+  orgId,
+}: ParticipantAvatarsProps) => {
+  return (
+    <Box alignItems="center" display="flex" flexWrap="wrap" gap={0.5}>
+      <AvatarGroup
```

Comment:
I had no idea about this component! I think it's great that you were able to find a component that takes care of the capping for us. 💯